.circleci/config.yml

```diff
@@ -273,21 +284,41 @@ references:
 
         [ -n "${NIGHTLY}" ] && IS_NIGHTLY=true || IS_NIGHTLY=false
         python3 ./Tests/scripts/collect_tests_and_content_packs.py -n $IS_NIGHTLY
+
   calculate_packs_dependencies: &calculate_packs_dependencies
     run:
       name: Calculate Packs Dependencies
       when: always
       command: |
         python3 ./Tests/Marketplace/packs_dependencies.py -i ./Tests/id_set.json -o $CIRCLE_ARTIFACTS/packs_dependencies.json
+
   update_tests_step: &update_tests_step
     run:
       name: Update Tests step
       when: always
       command: |
         python3 ./Tests/scripts/update_conf_json.py
         cp "./Tests/conf.json" "$CIRCLE_ARTIFACTS/conf.json"
+
+  get_contribution_pack: &get_contribution_pack
+    when:
+      condition: << pipeline.parameters.contrib_branch >>
```

Comment:
should we use `CONTRIB_BRANCH` here ?

Reply:
We set this env variable in the `environment` alias step - if a job doesn't use this alias for setting the environment, then the variable will not be set. I think it's preferable to use the pipeline parameter since that should always be present.